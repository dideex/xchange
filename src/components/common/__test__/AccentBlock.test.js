import React from 'react'
import Component from '../AccentBlock'
import {shallow} from 'enzyme'

describe('AccentBlock behaviour', () => {
  it('Markup', () => {
    const wrapper = shallow(<Component value="snapshot" />)
    expect(wrapper.html()).toMatchSnapshot()
  })

  it('Value should be', () => {
    const wrapper = shallow(<Component value="testValue" />)
    expect(wrapper.html()).toContain('testValue')
  })
  
  it('Value should be', () => {
    const wrapper = shallow(<Component value="testValue 2" />)
    expect(wrapper.html()).toContain('testValue 2')
  })
})
