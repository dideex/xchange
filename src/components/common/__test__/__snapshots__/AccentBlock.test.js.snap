// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccentBlock behaviour Snapshot component 1`] = `"<div class=\\"css-1oingry\\"><div class=\\"css-gudk1o\\">snapshot</div></div>"`;
